### DISCUSSION TITLE
Reporting a comment

### DISCUSSION THREAD

**Discussion by Michael21107** [community/community#77733] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

if i click on the 3 dots in a pull request comment, i cant see any report option

**Comment by MantasTech** [comment]
Hey Michael21107, 

Try contacting support with a screenshot of the comment for help.

**Reply by Michael21107** [reply]
i tried but everything redirects me to either forum or some doc
